UI/UX

jaketycyn opened the pull request:
Created Themes + Theme Changing element.

 Integrated 4 themes (Dark, White, Original & Vaal) as our 'base' themes w/ Dark theme being the default. Original holds most of the original colors you had with only some slight changes to things like hovers within selects.

Lots of small fixes like margins, focus, rings, centering, etc.

zach-herridge commented:
Hey this looks really good and it's super cool being able to switch themes! I have 3 small things:

1. Not sure if this is some sort of cache issue but the logo background color looks a bit weird for me
![image](https://user-images.githubusercontent.com/6809142/219546327-4deda6e1-1417-442d-b5d1-a6333ad73d6d.png)
2. These are getting cut off for me is that happening when you run it locally?
![image](https://user-images.githubusercontent.com/6809142/219546376-0fff1714-48e3-40e7-8ce9-f5c6a9f540be.png)
3. Not sure about this but the page scroll bar feels a bit more choppy than it did before

jaketycyn commented:
- Fixed the logo, 
- Changed Scrollbar to a more neutral color
- Removed light mode for now, but we can add it back in the future if it's requested.
